src/sentry/migrations/0706_grouphistory_userteam_backfill.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 5.0.4 on 2024-04-25 19:58
+
+from django.apps.registry import Apps
+from django.db import migrations
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from progressbar.utils import logging
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBarApprox
+
+
+def backfill_grouphistory_actor(apps: Apps, schema_editor: BaseDatabaseSchemaEditor) -> None:
+    GroupHistory = apps.get_model("sentry", "GroupHistory")
+    Actor = apps.get_model("sentry", "Actor")
+
+    query = GroupHistory.objects.filter(actor_id__isnull=False)
+    for history in RangeQuerySetWrapperWithProgressBarApprox(query):
```

Comment:
The `Approx` wrapper doesn't work when you're using a filter, it just estimates the whole size of the table. Given that the table is 400m rows, I think it might be worth just iterating the whole thing here, since we don't have an index on `actor_id, id`.

Alternatively, since there are only 11m rows to update, maybe we just use `RangeQuerySetWrapper` and don't worry about the progress bar? Still not totally sure how well the query will run though